modules/otel-collector-components.adoc

```diff
@@ -14,9 +14,9 @@ Receivers get data into the Collector.
 [id="otlp-receiver_{context}"]
 === OTLP Receiver
 
-The OTLP receiver ingests traces and metrics using the OpenTelemetry protocol (OTLP).
+The OTLP Receiver ingests traces and metrics using the OpenTelemetry protocol (OTLP).
```

Comment:
```suggestion
The OTLP Receiver ingests traces and metrics by using the OpenTelemetry protocol (OTLP).
```
Good to have (If you did care to fix this)

Reply:
Thank you. TBH, I normally include `by` in such phrases. However, here the OTLP is not a mean but rather an attribute. So it seemed to appropriate for the intended meaning to have `by` omitted.